<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMFB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="nextStepButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>48</y>
      <width>101</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preStepButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>94</y>
      <width>86</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Pre Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stateButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>80</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AutoButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>86</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto:OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RestartButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>86</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>ReStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WashButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>121</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Wash:OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
